src/libraries/System.Buffers/ref/System.Buffers.csproj

```diff
@@ -6,10 +6,11 @@
     <Configurations>$(NetCoreAppCurrent)-Debug;$(NetCoreAppCurrent)-Release;netstandard2.0-Debug;netstandard2.0-Release</Configurations>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetsNetCoreApp)' != 'true'">
```

Comment:
Agree (not in this PR).

> why does this one have a netstandard2.0 configuration

It is to make this hack work: https://github.com/dotnet/runtime/blob/master/src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.csproj#L5 . (I do not really understand what this hack does or whether it is still needed.)

Reply:
It was due to https://github.com/dotnet/corefx/issues/38729.  /cc @joperezr 

Rather than use a ProjectReference we can just restore the package for System.Buffers.  Using a ProjectReference here is dangerous since we aren't even shipping that version of System.Buffers.

Reply:
Opened #2257